src/components/molecules/ZopaFooter/styles/index.ts

```diff
@@ -0,0 +1,9 @@
+export { Footer } from './Footer';
```

Comment:
not sure you need to nest everything under `styles`? technically, these are still components

Reply:
I can change the naming of the folder, as you prefer.